crates/core/src/action.rs

```diff
@@ -112,11 +113,12 @@ pub enum Action {
     // DHT actions:
     // -------------
     /// Adds a holding workflow to the queue.
-    QueueHoldingWorkflow(PendingValidation),
+    /// With optional delay where the SystemTime is the time when the action got dispatched
```

Comment:
can we add a line of comment here explaining the use of the PendingValidation struct is for here?